extensions/wikia/EditPageLayout/js/editor/AceEditor.js

```diff
@@ -176,6 +177,19 @@ define(
 		$('.editpage-widemode-trigger').click(editorModeChange);
 	}
 
+	function launchEditorPreview(){
+		var editorValue = ace.getContent(),
+			wikiaDomain = mw.config.get('wgServer').split('://')[1],
```

Comment:
What's the benefit of it? We know for sure that wgServer is correct - we control it (we set it, we depend on it in many places).